test/cpp/naming/resolver_component_tests_runner_invoker.cc

```diff
@@ -123,24 +133,47 @@ int main(int argc, char** argv) {
         bin_dir + "/resolver_test_record_groups.yaml",
         bin_dir + "/utils/dns_resolver", bin_dir + "/utils/tcp_connect");
 #else
-    grpc::testing::ManifestFile manifest_file(
-        grpc::testing::NormalizeFilePath(test_srcdir.value()) + "\\MANIFEST");
-    grpc::testing::InvokeResolverComponentTestsRunner(
-        grpc::testing::NormalizeFilePath(
-            manifest_file.Get("com_github_grpc_grpc/test/cpp/naming/"
-                              "resolver_component_tests_runner.exe")),
-        grpc::testing::NormalizeFilePath(
-            manifest_file.Get("com_github_grpc_grpc/test/cpp/naming/" +
-                              absl::GetFlag(FLAGS_test_bin_name) + ".exe")),
-        grpc::testing::NormalizeFilePath(manifest_file.Get(
-            "com_github_grpc_grpc/test/cpp/naming/utils/dns_server.exe")),
-        grpc::testing::NormalizeFilePath(
-            manifest_file.Get("com_github_grpc_grpc/test/cpp/naming/"
-                              "resolver_test_record_groups.yaml")),
-        grpc::testing::NormalizeFilePath(manifest_file.Get(
-            "com_github_grpc_grpc/test/cpp/naming/utils/dns_resolver.exe")),
-        grpc::testing::NormalizeFilePath(manifest_file.Get(
-            "com_github_grpc_grpc/test/cpp/naming/utils/tcp_connect.exe")));
+    if (absl::GetFlag(FLAGS_running_locally)) {
```

Comment:
there's a lot of copy/paste between the two possible option values. Sounds like you should have a utility function to locate a runfile and then you won't have the 2 invocations of grpc::testing::InvokeResolverComponentTestsRunner( with almost exactly the same params (and repeating each path twice).

Also, it sounds like accessing runfiles is something that we've must have done previously in other tests?

Reply:
Done. not needed anymore.